vm/src/stdlib/os.rs

```diff
@@ -3122,14 +3135,20 @@ mod minor {
 
     pub const SYMLINK_DIR_FD: bool = false;
 
-    #[pyfunction]
-    pub(super) fn symlink(
-        _src: PyPathLike,
-        _dst: PyPathLike,
+    #[derive(FromArgs)]
+    pub(super) struct SimlinkArgs {
+        #[pyarg(any)]
+        src: PyPathLike,
+        #[pyarg(any)]
+        dst: PyPathLike,
+        #[pyarg(flatten)]
         _target_is_directory: TargetIsDirectory,
```

Comment:
`_` prefix in struct will affact keyword argument name.

Reply:
Since `_target_is_directory` is `#[pyarg(flatten)]`, `_` prefix will not affect keyword argument name.

```rust
#[derive(FromArgs, Default)]
pub struct TargetIsDirectory {
    #[pyarg(any, default = "false")]
    target_is_directory: bool,
}
```